bindings/rust/src/lib.rs

```diff
@@ -20,10 +20,21 @@ pub struct Module(*const sys::FizzyModule);
 
 impl Drop for Module {
     fn drop(&mut self) {
+        debug_assert!(!self.0.is_null());
```

Comment:
Changed these to `debug_assert` and added it to all relevant places. A failure of this would mean a programming error in our code, and not user code. Hence `debug_assert`, i.e. it will be caught by our tests assuming we have high enough coverage.